protocol/x/bridge/keeper/acknowledge_bridges.go

```diff
@@ -76,28 +75,33 @@ func (k Keeper) GetAcknowledgeBridges(
 	}
 }
 
-// AcknowledgeBridges acknowledges a list of bridge events.
+// AcknowledgeBridges acknowledges a list of bridge events and returns an error if any of following
+// - bridging is disabled.
+// - fails to delay a `MsgCompleteBridge` for any bridge event.
+// - fails to upate `AcknowledgedEventInfo` in state.
 func (k Keeper) AcknowledgeBridges(
 	ctx sdk.Context,
 	bridgeEvents []types.BridgeEvent,
 ) (err error) {
+	if len(bridgeEvents) == 0 {
+		return nil
+	}
+	safetyParams := k.GetSafetyParams(ctx)
```

Comment:
Wasn't this disabling logic part of a previous PR? Is there a test case for this below?